rasa/core/policies/ted_policy.py

```diff
@@ -287,25 +287,32 @@ def _create_model_data(
 
         X_sparse = []
         X_dense = []
+        X_entities = []
 
         for dial in data_X:
             sparse_state = []
             dense_state = []
+            entities = []
             for state in dial:
                 if state[0] is not None:
                     sparse_state.append(state[0].astype(np.float32))
                 if state[1] is not None:
                     dense_state.append(state[1])
+                if state[2] is not None:
+                    entities.append(state[2])
 
             if not sparse_state == []:
                 sparse_state = scipy.sparse.vstack(sparse_state)
             if not dense_state == []:
                 dense_state = np.vstack(dense_state)
+            if not entities == []:
+                entities = np.vstack(entities)
             X_sparse.append(sparse_state)
             X_dense.append(dense_state)
-
+            X_entities.append(entities)
```

Comment:
if entities go separately, where are slots?

Reply:
they're on a separate branch for now :) I'll add them after we merge this because slots branch used old data format, without passing message directly.